src/transformers/models/whisper/tokenization_whisper.py

```diff
@@ -692,6 +716,9 @@ def _decode(
         if normalize:
             clean_text = self._normalize(text)
             return clean_text
+        elif basic_normalize:
```

Comment:
I guess we don't save this to `self.basic_normalize` because only used for training / evaluation maybe?

Reply:
That's a good point - I followed the convention we used for `normalize` where we load the normaliser on init, but don't save any settings. Do you think it would be worth having a 'saved' normaliser? I don't think it's going to be a super popular use-case since the base normalisers are pretty good already, and the normaliser is **only** used for academic evaluation. Can do in a follow-up PR if you think so though